NuCivic/dkan
# Make sidebar styles more specific so they don't leak into admin pages CIVIC-4949

Issue: CIVIC-4949

## Description

Sidebar header styles interfere with the usability of the page manager admin screen.
![screen shot 2016-11-23 at 11 07 53 am](https://cloud.githubusercontent.com/assets/314172/20770867/caf315b2-b70c-11e6-8978-afa752fe6007.png)


## How to reproduce

1. Navigate to a page manager admin screen `admin/structure/pages/edit/page-datasets`
2. Click 'Content' on one of the variants
3. Notice the titles are picking up styles from h2.panel-title

## QA Tests

1. Navigate to a page manager admin screen `admin/structure/pages/edit/page-datasets`
2. Click 'Content' on one of the variants
3. Notice the titles are NOT picking up styles from h2.panel-title or .ctools-collapsible-container
